src/test/unit_tests/test_bucketspace_versioning.js

```diff
@@ -378,35 +379,753 @@ mocha.describe('bucketspace namespace_fs - versioning', function() {
             const comp_res = await compare_version_ids(full_path, key, res.VersionId);
             assert.ok(comp_res);
         });
+
+
+        mocha.it('delete object version id - disabled key', async function() {
```

Comment:
I didn't go over all the tests one by one... but in general it seems you didn't put any comments in it, maybe you will want to add some. at least about what you are doing, what you expect and things like that.